loadgen/helpers.go

```diff
@@ -0,0 +1,34 @@
+package loadgen
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/sdk/client"
+)
+
+func VisibilityCountIsEventually(
```

Comment:
Arguably this is a tiny little thing only needed by one test and doesn't deserve it's own file/function. This is overabstraction IMO and will lead to sprawl.

Reply:
Like the other thing, there's almost certainly going to be things like this in the future, I can't imagine we're never going to share any common stuff ever

Reply:
We will share when we share, but this is a bit too premature IMO